tracer/build/_build/Build.cs

```diff
@@ -30,8 +30,8 @@ partial class Build : NukeBuild
     [Parameter("Configuration to build - Default is 'Release'")]
     readonly Configuration BuildConfiguration = Configuration.Release;
 
-    [Parameter("Platform to build - x86 or x64. Default is x64")]
-    readonly MSBuildTargetPlatform TargetPlatform = MSBuildTargetPlatform.x64;
+    [Parameter("Platform to build - x86, x64, ARM64. Default is x64")]
```

Comment:
This comment is not correct anymore.
```suggestion
    [Parameter("Platform to build - x86, x64, ARM64. Defaults to the current platform.")]
```